utils/dc-chain/patches/newlib-3.3.0-kos.diff

```diff
@@ -135,7 +135,7 @@ diff -ruN newlib-3.3.0/newlib/libc/include/sys/_pthreadtypes.h newlib-3.3.0-kos/
 -
 -typedef struct {
 -  int      is_initialized;
--  clock_t  clock;             /* specifiy clock for timeouts */
+-  clock_t  clock;             /* specify clock for timeouts */
```

Comment:
No diff file should be edited as it would break the diffing.

Reply:
Great POINT!